docs/contributor_guide/how-tos/release.md

```diff
@@ -2,47 +2,83 @@
-in GitHub Actions.
-Follow the linked instructions.
+We use [`jupyter_releaser`](https://jupyter-releaser.readthedocs.io/en/latest/) to
+create our releases in GitHub Actions.
+This document is a quick reference that will work for most releases.
+For full detailed instructions, see the
+[`jupyter_releaser` "Making your first release" document](https://jupyter-releaser.readthedocs.io/en/latest/get_started/making_release_from_repo.html).
 
+:::{important}
 **This project uses [Semantic Versioning](https://semver.org)**.
+:::
+
+## Prerequisites
+
+- To do the following steps, you must have "admin" privileges on the repository.
+
+## Step 1a: Prep release
+
+**This step will bump versions, update the changelog, and create a "draft" release in
+GitHub.**
+
+- From the [JupyterGIS actions menu](https://github.com/geojupyter/jupytergis/actions),
+  select "Step 1: Prep Release" action from the left pane.
+- On the right, click "Run workflow". This will present a menu you need to fill out.
+- The only thing you normally need to input here is the "new version specifier".
+  **The default value is `next`, but we recommend always specifying a numeric specifier, e.g. `0.3.0`**.
+
+  :::{danger}
+  **Specifying a version part (e.g. `minor` or `patch`) for "New Version Specifier" will
+  cause Step 2 to fail.**
+
+  Read below for more important information about the version specifier.
+  :::
+
+### Version specifier
+
+Ensure you understand the [Semantic Versioning](https://semver.org) version part
+definitions ("major", "minor", "patch") before selecting a version.
 
 :::{important}
-Because this project is in early development, we **do not bump the major version number**.
-Most changes are minor version bumps, even breaking changes.
-See [the SemVer FAQ](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) for more.
+Because this project is currently in early development (pre-1.0), we **do not bump the
+major version number**.
+
+This means that **even breaking changes are minor version bumps**.
+
+See [the SemVer FAQ](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase) for more details.
 :::
 
-### Specifying a version spec
+## Step 1b: Review changelog
 
-When prompted for a "New Version Specifier", the default value is `next`.
-This will bump the packages as follows:
+**This step will make the release notes more readable for end-users.**
 
-- `0.1.0a0` -> `0.1.0a1`
-- `0.1.0b7` -> `0.1.0b8`
-- `0.1.0` -> `0.1.1`
+- Visit the new [GitHub "draft" release](https://github.com/geojupyter/jupytergis/releases) created in Step 1a.
+- Edit the release text to fix any typos and make other edits for end-user
+  accessibility.
+  - Remove any bot-created PRs, for example pre-commit hook updates or dependabot PRs.
+  - Remove any bots from the contributor list.
+  - Edit text for readability by end-users where appropriate.
```

Comment:
```suggestion
  - Edit the text to improve readability for end-users where appropriate.
```

Reply:
I think its ok the other way too.

Reply:
This is a great change, thanks :)

Reply:
Woops I had auto-merge on so this didn't go in.